Tiltfile

```diff
@@ -388,6 +388,128 @@ def prepare_all():
     )
     local(cmd, env = settings.get("kustomize_substitutions", {}))
 
+# create cluster template resources from cluster-template files in the templates directory
```

Comment:
Let's not change the Tiltfile in this PR. Folks are just using Tiltfile from main usually and we don't cherry-pick any Tilt changes